company-profiles/basisPointDesignLtd.md

```diff
@@ -0,0 +1,25 @@
+# basispointdesignltd
+
+## Company blurb
+
+Basis Point Design is a brand, web and marketing communications specialist that services emerging and established investment managers across the globe.
+
+## Company size
+
+11-50 employees
+
+## Remote status
+
+Branding, Graphic Design, Marketing Presentations, Website Design, Website Development, Marketing Communications, Investor Communications, Pitch Deck Design, Digital Design, Investor Portal Design and Build, Marketing Emailers, Video Production, Animation, Infographics, UI UX Design, Infographics, 3D Motion, Marketing Videos, Audio Pitch Decks, Fact Sheet Design, and Investor Reporting
```

Comment:
This copy has nothing to do with the remote status. As mentioned [in the example](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md#remote-status), this should related to the remote culture of the company.